modules/3d/misc/python/test/test_volume.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import cv2 as cv
+
+from tests_common import NewOpenCVTests
+
+class volume_test(NewOpenCVTests):
+    def test_VolumeDefault(self):
+        depth = self.get_sample('cv/rgbd/depth.png', cv.IMREAD_ANYDEPTH).astype(np.float32)
+
+        Rt = np.array(
+            [[1, 0, 0, 0],
+            [0, 1, 0, 0],
+            [0, 0, 1, 0],
+            [0, 0, 0, 1]])
+        volume = cv.Volume()
+        volume.integrate(depth, Rt)
+
+        size = (480, 640, 4)
+        points  = np.zeros(size, np.float32)
+        normals = np.zeros(size, np.float32)
+
+        Kraycast = np.array([[525. ,   0. , 319.5],
+                             [  0. , 525. , 239.5],
+                             [  0. ,   0. ,   1. ]])
+
+        volume.raycastEx(Rt, size[0], size[1], Kraycast, points, normals)
+        volume.raycast(Rt, points, normals)
```

Comment:
Is there simple way to check, if volume was modified by the calls to be sure that proper method was called?

Reply:
Didn't get it; the only method that modifies the volume is `integrate/integrateColor()`
All volumes support `raycast/raycastEx` but only ColorTSDF supports `*color()` methods